packages/contracts/contracts/protocol/lendingpool/LendingPool.sol

```diff
@@ -711,6 +714,30 @@ contract LendingPool is
         _reserves[asset][trancheId].configuration.data = configuration;
     }
 
+    /**
+     * @dev Sets the configuration bitmap of the reserve as a whole
+     * - Only callable by the LendingPoolConfigurator contract
+     * @param asset The address of the underlying asset of the reserve
+     * @param trancheId The tranche id of the reserve
+     * @param ltv The new ltv
+     * @param liquidationThreshold The new liquidationThreshold
+     * @param liquidationBonus The new liquidationBonus
+     * @param borrowFactor The new borrowFactor
+     **/
+    function setCollateralParams(
+        address asset,
+        uint64 trancheId,
+        uint256 ltv,
+        uint256 liquidationThreshold,
```

Comment:
set all eligible param types to uint64 to avoid converting on-chain

Reply:
done